src/core/ext/filters/client_channel/lb_policy/weighted_round_robin/weighted_round_robin.cc

```diff
@@ -424,8 +425,8 @@ void WeightedRoundRobin::AddressWeight::MaybeUpdateWeight(
   if (GRPC_TRACE_FLAG_ENABLED(grpc_lb_wrr_trace)) {
     gpr_log(GPR_INFO,
             "[WRR %p] subchannel %s: qps=%f, eps=%f, cpu_utilization=%f "
-            "error_util_penalty=%f : setting weight=%f old_weight=%f now=%s "
-            "last_update_time=%s non_empty_since=%s",
+            "error_util_penalty=%f : setting weight=%f weight_=%f now=%s "
+            "last_update_time=%s non_empty_since_=%s",
```

Comment:
Still need to re-add the underscore on `last_update_time`.

Reply:
Done.